src/frontend/js/widgets/Dashboard/components/DashboardSidebar/_styles.scss

```diff
@@ -22,6 +22,9 @@
       &__avatar {
         position: absolute;
         top: calc($avatar-size / -2);
+        &.dashboard__avatar__square {
+          top: calc($avatar-size / 3 * -2);
+        }
```

Comment:
I wonder if those styles should not be moved into a `_styles.scss` file next to DashboardAvatar component ?

Reply:
you're right, it was the last bit of dashboard that had to be moved.
done.